src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,45 @@
+package duke;
+
+/**
+ * A parser object that makes sense of all the user input.
+ */
+
+public class Parser {
+
+    Parser() { }
+
+    /**
+     * split up the user input and recognise the executable functions
```

Comment:
Should be "Splits", note the tense and capitalisation, also expected full stop at the end